app/scripts/lib/ComposableObservableStore.js

```diff
@@ -0,0 +1,49 @@
+const ObservableStore = require('obs-store')
```

Comment:
I'm sorry we did not include this on the issue, but something like this existed already:
https://github.com/kumavis/obs-store/blob/master/lib/composed.js

That said, you have more features here, so it's probably better to keep this.

Reply:
Your time zone really incentivizes me waking up early and starting to look at code right away! You might turn me into an earlier riser!